db/migrations/20220601183937_create_asg_lastest_updates.rb

```diff
@@ -1,7 +1,7 @@
 Sequel.migration do
   change do
     create_table :asg_timestamps do
-      primary_key name: :id
+      primary_key :id, name: :id
```

Comment:
I think enough time has passed that we don't need to worry about someone having run this migration, but not the followup that corrected it.  With Ruby 3.4, this migration would no longer behave consistently, anyway (`name: id` get serialized to `{name: :id}` instead of `{:name=>:id}`

Reply:
I initially tried to make these migrations more explicit, but that caused issues with the `rename_column` on MySQL 5.7 tests.  Which we don't actually support, so a different potential workaround is dropping those tests.

Reply:
Is it worth just dropping [5.7 from the pipeline](https://github.com/cloudfoundry/cloud_controller_ng/blob/main/.github/workflows/unit_tests.yml#L79) then?